x-pack/plugins/infra/server/lib/alerting/metric_threshold/register_metric_threshold_rule_type.ts

```diff
@@ -110,6 +116,12 @@ export async function registerMetricThresholdRuleType(
         { name: 'metric', description: metricActionVariableDescription },
         { name: 'threshold', description: thresholdActionVariableDescription },
         { name: 'viewInAppUrl', description: viewInAppUrlActionVariableDescription },
+        { name: 'cloud', description: cloudActionVariableDescription },
```

Comment:
What would be the best order for elements in the dropdown list to be displayed to the user?

As in: if you are a user and you're hunting through this list, how could most easily find what you're looking for? If the list was alphabetised, or perhaps first categorised and then alphabetised?

Reply:
I think alphabetised would be easier to scroll through and find the element.

If I remember correctly, @maciejforcone had some ideas on improving this part of the UI.